### PR TITLE
Update use-dkim-to-validate-outbound-email.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a line to the third party services stating that some of the 3rd party servers can have their own CName records

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3398

File: microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md [hunk 1]
```diff
@@ -306,7 +306,7 @@ In this example, the host name and domain contain the values to which the CNAME
 ## Set up DKIM so that a third-party service can send, or spoof, email on behalf of your custom domain
 <a name="SetUp3rdPartyspoof"> </a>
 
-Some bulk email service providers, or software-as-a-service providers, let you set up DKIM keys for email that originates from their service. This requires coordination between yourself and the third-party in order to set up the necessary DNS records. No two organizations do it exactly the same way. Instead, the process depends entirely on the organization.
+Some bulk email service providers, or software-as-a-service providers, let you set up DKIM keys for email that originates from their service. This requires coordination between yourself and the third-party in order to set up the necessary DNS records. Some third-party server can have it's own CNAME records with different selectors. No two organizations do it exactly the same way. Instead, the process depends entirely on the organization.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md]:
```suggestion
Some bulk email service providers, or software-as-a-service providers, let you set up DKIM keys for email that originates from their service. This requires coordination between yourself and the third-party in order to set up the necessary DNS records. Some third-party servers can have their own CNAME records with different selectors. No two organizations do it exactly the same way. Instead, the process depends entirely on the organization.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**